stdpopsim/genomes.py

```diff
@@ -542,10 +528,14 @@ def add_single_site(
 
         mut_type = stdpopsim.MutationType(
             distribution_type="f",
-            dominance_coeff=dominance_coeff,
-            distribution_args=[selection_coeff],
+            dominance_coeff=0.5,
+            distribution_args=[0.0],
             convert_to_substitution=False,
         )
+
+        # TODO: Is there a need to safeguard against `_recap_and_rescale` simulating
+        # mutations at the single site, because it is recorded as strictly neutral?
```

Comment:
annoyingly, yes, we should probably do this

Reply:
OK, I'll figure out a way to stick the selection/dominance coefficients into the metadata for the parent DFE.